### PR TITLE
Week commencing 16th may

### PR DESCRIPTION
**Pull request by springcoil**
* Updating the changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,109 @@
...
+ networks. 
+
+* We've updated our [voice library page](https://library.api.audio/voices) to have faster loading and we've added some other improvements in the GET request for voices so you can have a better voice discovery experience. Most of this is backend and performance enhancements. 
+* Enhanced query parameters for voice discoverability
+
+`GET https://v1.api.audio/voice?tags=upbeat,storytelling`
+
+Will voices that have both of these tags. We also introduced pagination as well. 
+
+**Chunking**
```
Comment (martinezpl) [inline CHANGELOG.md:35]:
```suggestion
**Pagination**
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,109 @@
...
+E.g.
+
+`GET https://v1.api.audio/voice?limit=10&offset=10`
+
+`GET https://v1.api.audio/voice?limit=10&offset=20`
+
+`GET https://v1.api.audio/voice?limit=10&offset=30`
+
+
+**Query body in POST /voice**
```
Comment (martinezpl) [inline CHANGELOG.md:48]: FYI This is not available in the SDK yet, only through HTTP request.

**Review by martinezpl — APPROVED** [review]
I redacted syncTTS description.

File: CHANGELOG.md (hunk 2 above)
Comment (springcoil) [reply CHANGELOG.md:48]: That's ok

**Pull request merged**